lib/services/ios-project-service.ts

```diff
@@ -347,7 +347,14 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				if(firstPostInstallIndex !== -1 && firstPostInstallIndex !== projectPodfileContent.lastIndexOf(IOSProjectService.PODFILE_POST_INSTALL_SECTION_NAME)) {
 					this.$logger.warn(`Podfile contains more than one post_install sections. You need to open ${this.projectPodFilePath} file and manually resolve this issue.`);
 				}
-
+				
+				let pbxprojFilePath = path.join(this.platformData.projectRoot, this.$projectData.projectName + IOSProjectService.XCODE_PROJECT_EXT_NAME, "xcuserdata");
+				if(!this.$fs.exists(pbxprojFilePath).wait()) {
+					this.$logger.info("Creating project scheme...");
+					let createScheme_rb = `echo \"require 'xcodeproj'; xcproj = Xcodeproj::Project.open('${this.$projectData.projectName}.xcodeproj'); xcproj.recreate_user_schemes; xcproj.save\" | ruby`;
```

Comment:
I've tested on a clean machine with only Xcode 6.4, node and nativescript installed. In my case, the following error was shown:
```
require "xcodeproj"
LoadError: cannot load such file -- xcodeproj
```

Do we need to install some additional dependencies?